Pod/Classes/NotificationConvenience.swift

```diff
@@ -17,6 +17,13 @@ public extension NotificationCenter {
         }
     }
     
+    // Needed for objc compatibility
+    public func post(notification: Notification) -> Void {
+        DispatchQueue.main.async { [weak self] in
```

Comment:
ah yeah now calling the main func. As for the bool, we always are calling on the main thread so that isnt needed for FPs sake